README.md

```diff
@@ -0,0 +1,65 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project 
+
+# Community Development 
+
+![screenshot](img/ScreenCap.png)
+
+>This website if a community development website dedicated for my local city
+
+>
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+
+## Live presentation
+
+- https://www.loom.com/share/8feee13007f2414182ba4ef53aee9173
+
+### Usage
+
+ - to get a local copy up and running follow these simple example steps.
+
+- Open terminal
+
+- Clone this project by the command `git clone https://github.com/mutinhiri/CapstonePr1`
```

Comment:
- Your Live Version isn't working.
- I tried to watch your video presentation but i just watched a black background in every moment. Kindly fix this.
![Screen Shot 2021-09-15 at 16 27 39](https://user-images.githubusercontent.com/22258274/133512442-ab1108f8-d626-4ea3-bff5-665abe8330d0.png)

Reply:
i am having problems trying to record my screen its recording black is it ok if i use another application